flowmachine/tests/test_histogram_aggregations.py

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from flowmachine.core import make_spatial_unit
+from flowmachine.features import (
+    ModalLocation,
+    daily_location,
+    SubscriberHandsetCharacteristic,
+)
+import numpy as np
+
+from flowmachine.features import Displacement, RadiusOfGyration, daily_location
+from flowmachine.features.utilities.histogram_aggregations import HistogramAggregation
+from flowmachine.features.subscriber.daily_location import locate_subscribers
+from flowmachine.utils import list_of_dates
+
+
+def test_create_histogram_using_int_bins_value(get_dataframe):
+    """
+    Create histogram using one bins value.
+    """
+    RoG = RadiusOfGyration("2016-01-01", "2016-01-02")
+
+    agg = HistogramAggregation(locations=RoG, bins=5)
+    df = get_dataframe(agg)
+    de = np.histogram(get_dataframe(RoG).value, bins=5)
+    assert set(de[0].tolist()) == set(df.value.tolist())
+
+
+def test_create_histogram_using_list_of_bins_values(get_dataframe):
+    """
+    Create histogram using list of bins values.
+    """
+    RoG = RadiusOfGyration("2016-01-01", "2016-01-02")
+
+    agg = HistogramAggregation(locations=RoG, bins=[10, 20, 30, 40, 50, 60])
+    df = get_dataframe(agg)
+    de = np.histogram(get_dataframe(RoG).value, bins=[10, 20, 30, 40, 50, 60])
+    assert set(de[0].tolist()) == set(df.value.tolist())
```

Comment:
Half way there :) Now just need to return the bin edges as in the numpy function, rather than the bucket number.